### PR TITLE
FEATURE: add Untranslated filter to admin text customization

### PR DESCRIPTION
**Pull request by renato**
Adds a checkbox to filter untranslated text strings in the admin UI, behind a hidden and default `false` site setting `admin_allow_filter_untranslated_text`.

### CODE REVIEW

**Review by gschlager — APPROVED** [review]
Looks good. I'd prefer a dropdown, but let's keep this PR simple.

**Comment by renato** [comment]
I think the dropdown will actually look worse in general (without the hidden site setting enabled), with only 2 options, and supporting it (in addition to the checkbox when disabled) seemed overkill.

File: lib/freedom_patches/translate_accelerator.rb [hunk 1]
```diff
@@ -80,6 +80,16 @@ def search(query, opts = {})
 
       if opts[:only_overridden]
         add_if_matches(overrides_by_locale(locale), results, regexp)
+      elsif opts[:only_untranslated]
```
[the remark this answers is not in the document]
Comment (renato) [reply lib/freedom_patches/translate_accelerator.rb]: Done!

**Comment by gschlager** [comment]
You are probably right. I'm not a designer. :wink: Feel free to merge.

**Pull request merged**